src/test/java/de/tum/in/www1/artemis/service/notifications/GeneralInstantNotificationServiceTest.java

```diff
@@ -0,0 +1,181 @@
+package de.tum.in.www1.artemis.service.notifications;
+
+import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.times;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import de.tum.in.www1.artemis.domain.User;
+import de.tum.in.www1.artemis.domain.enumeration.GroupNotificationType;
+import de.tum.in.www1.artemis.domain.enumeration.NotificationType;
+import de.tum.in.www1.artemis.domain.notification.GroupNotification;
+import de.tum.in.www1.artemis.domain.notification.Notification;
+import de.tum.in.www1.artemis.domain.notification.NotificationConstants;
+import de.tum.in.www1.artemis.service.notifications.push_notifications.ApplePushNotificationService;
+import de.tum.in.www1.artemis.service.notifications.push_notifications.FirebasePushNotificationService;
+
+class GeneralInstantNotificationServiceTest {
+
+    @Mock
+    private NotificationSettingsService notificationSettingsService;
+
+    @Mock
+    private ApplePushNotificationService applePushNotificationService;
+
+    @Mock
+    private FirebasePushNotificationService firebasePushNotificationService;
+
+    @Mock
+    private MailService mailService;
+
+    private User student1;
+
+    private User student2;
+
+    private GeneralInstantNotificationService generalInstantNotificationService;
+
+    private Notification notification;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
Apple/Firebase notification service use `openMocks` instead. Where is the difference?